frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -172,16 +172,17 @@ function FileManagerMenu:setUpdateItemTable()
                 sub_item_table = {
                     {
                         text_func = function()
-                            return T(_("Items per page: %1"), G_reader_settings:readSetting("items_per_page"))
+                            return T(_("Items per page: %1"),
+                                G_reader_settings:readSetting("items_per_page") or FileChooser.items_per_page_default)
```

Comment:
Like that or like
```suggestion
                                G_reader_settings:readSetting("items_per_page", FileChooser.items_per_page_default)
```

Reply:
Let's save the setting when a user changes it only. No need to save the hard-coded default value
https://github.com/koreader/koreader/blob/601925ab87bc7538eb96b8f32de9b4e37bfc676d/frontend/ui/widget/menu.lua#L622